src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionsCallout.styles.ts

```diff
@@ -0,0 +1,130 @@
+import {
+    FontSizes,
+    FontWeights,
+    IStyle,
+    ITheme,
+    mergeStyleSets
+} from '@fluentui/react';
+import {
+    IConditionsCalloutStyleProps,
+    IConditionsCalloutStyles
+} from './ConditionsCallout.types';
+
+export const classPrefix = 'cb-conditionscallout';
+const classNames = {
+    root: `${classPrefix}-root`,
+    footer: `${classPrefix}-footer`,
+    title: `${classPrefix}-title`
+};
+export const getStyles = (
+    _props: IConditionsCalloutStyleProps
+): IConditionsCalloutStyles => {
+    return {
+        root: [classNames.root],
+        footer: [
+            classNames.footer,
+            {
+                paddingTop: 32
+            }
+        ],
+        title: [
+            classNames.title,
+            {
+                fontSize: FontSizes.size16,
+                margin: 0
+            }
+        ],
+        subComponentStyles: {
+            callout: {
+                root: {
+                    width: 300,
+                    padding: 20
+                }
+            }
+        }
+    };
+};
+
+export const getSummaryStyles = (theme: ITheme) =>
+    mergeStyleSets({
+        title: {
+            margin: 0
+        },
+        invalidText: {
+            color: theme.semanticColors.errorText,
```

Comment:
I think we actually usually use one of our scss variable for error text. This might match fine but might be worth checking other places we show errors. I didn't actually know they had this semantic color so this is good to know about. Let's just make sure it's consistent.

Reply:
That should be a good red then. let's just make sure it hits contrast ratios.